src/client/components/Story/Body.js

```diff
@@ -97,29 +100,66 @@ export function getHtml(body, jsonMetadata = {}, returnType = 'Object', options
   return <div dangerouslySetInnerHTML={{ __html: sections.join('') }} />;
 }
 
+const getObjectDetails = metaData => {
+  try {
+    if (!metaData) return {};
+    const parsedJsonMetadata = jsonParse(metaData) || {};
+    return get(parsedJsonMetadata, 'waivioRewards.proposition') || {};
```

Comment:
use array for prop path: get(parsedJsonMetadata, ['waivioRewards', 'proposition'])